src/pages/BiddingPage/BiddingPage.js

```diff
@@ -0,0 +1,462 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function BiddingPage() {
+  const [pageMode, setPageMode] = useState('판매');
+  const [buttonMode, setButtonMode] = useState([false, true]);
+  const [modeChange, setModeChange] = useState(true);
```

Comment:
```suggestion
  const [currentBtn, setCurrentBtn] = useState(1);
```

이 두가지 작업을 하나의 state로 관리할 수 있겠습니다.